configure.py

```diff
@@ -444,8 +444,6 @@ def process_command_line(args): # pylint: disable=too-many-locals
 
     install_group.add_option('--prefix', metavar='DIR',
                              help='set the install prefix')
-    install_group.add_option('--destdir', metavar='DIR',
```

Comment:
Concur, I'm ok with this change generally but we can't just remove `--destdir` completely as all current downstream packagers likely use it and we should at least let them know at build time what needs to change in their scripts.

Alternately we could keep `--destdir` and accept env DESTDIR as an override, but it seems autoconf only uses the env variable for this, and removing the flag simplifies everything a bit.